src/pages/vrf/v2/lootbox.mdx

```diff
@@ -0,0 +1,267 @@
+---
+layout: ../../../layouts/MainLayout.astro
+section: vrf
+title: "Create a lootbox to distribute random rewards"
+---
+
+import { Aside } from "@components"
+import { Tabs } from "@components/Tabs"
+
+The lootbox is a utility contract that allow users to open it to receive a random reward.
+
+It supports [ERC-20](https://eips.ethereum.org/EIPS/eip-20), [ERC-721](https://eips.ethereum.org/EIPS/eip-721), and [ERC-1155](https://eips.ethereum.org/EIPS/eip-1155) tokens as rewards. The rewards are distributed from a pool of tokens that are transferred to the contract when the lootbox contract is deployed.
+
+The lootbox contract uses VRF to generate a random number that is used to determine the reward.
+
+<Aside type="caution" title="Disclaimer">
+  This repository represents an example of using a Chainlink product or service. It is provided to to help you
+  understand how to interact with Chainlink's systems so that you can integrate them into your own. This template is
+  provided "AS IS" without warranties of any kind, has not been audited, and may be missing key checks or error handling
+  to make the usage of the product more clear. Take everything in this repository as an example and not something to be
+  copy pasted into a production ready service.
+</Aside>
+
+## Before you begin
+
+Before you start this tutorial, complete the following items:
+
+- Install [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+  - To check your installation, run `git --version`. You should see an output similar to `git version x.x.x`.
+- Install [Nodejs](https://nodejs.org/en/) 16.0.0 or higher
+  - To check your installation, run `node --version`. You should see an output similar to `v16.x.x`.
+- Make sure the deployer account has enough LINK to fund the [VRF subscription](/vrf/v2/subscription). If you don't already have a VRF subscription, the deployment script creates one for you.
+  - [Estimate the minimum subscription balance](/vrf/v2/estimating-costs#estimate-minimum-subscription-balance) that VRF requires to process your randomness request. The minimum subscription balance provides a buffer against gas volatility, and only the actual cost of your request will be deducted from your account. If your subscription is underfunded, your VRF request will be pending for 24 hours. If that happens, check the [Subscription Manager](https://vrf.chain.link]) to see the additional balance needed.
+  - Configure the initial funding amount in [`network-config.ts`](https://github.com/smartcontractkit/quickstarts-lootbox/blob/main/network-config.ts).
+  - For testnets, you can use the [LINK faucet](https://faucets.chain.link/). Request 20 LINK from the faucet - this amount should be sufficient to run this tutorial with a testnet.
```

Comment:
```suggestion
  - To deploy this contract on testnets, use the [LINK faucet](https://faucets.chain.link/) to retrieve testnet LINK. Request 20 LINK from the faucet - this amount should be sufficient to run this tutorial on a testnet.
```

Reply:
Shall we also add this: https://docs.chain.link/resources/acquire-link?parent=gettingStarted